docs/developers/build-starrocks/build_starrocks_on_ubuntu.md

```diff
@@ -0,0 +1,52 @@
+### Prerequisite
+
+```
+sudo apt-get install automake binutils-dev bison byacc ccache flex libiberty-dev libtool maven zip python3 python-is-python3 -y
+```
+
+### Compiler
+If the ubuntu version >= 22.04, you can
+```
+sudo apt-get install cmake gcc g++ default-jdk -y
+```
+
+If the ubuntu version < 22.04.
+Check the version of following tools and compilers
+
+##### 1. GCC/G++
+GCC/G++ version must be >= 10.3
+```
+gcc --version
+g++ --version
+```
+Install GCC/G++(https://gcc.gnu.org/releases.html)
+
+##### 2. JDK
+OpenJDK version must be >= 8
```

Comment:
It should emphasize that JDK is required, not JRE, and tell people how to identify which is which.